src/azure-cli/azure/cli/command_modules/vm/_params.py

```diff
@@ -517,6 +519,10 @@ def load_arguments(self, _):
         c.argument('image_location', get_location_type(self.cli_ctx))
         c.argument('edge_zone', edge_zone_type)
 
+    with self.argument_context('mask') as c:
```

Comment:
Does this part required for this change?

Reply:
No so deleted this part